_publications/2025-interactive-data-augmentation.html

```diff
@@ -0,0 +1,25 @@
+---
+layout: publication
+year: 2025
+title: "Exploring Empty Spaces: Human-in-the-Loop Data Augmentation"
+authors:
+  - Catherine Yeh
+  - Donghao Ren
+  - Yannick Assogba
+  - Dominik Moritz
+  - Fred Hohman
+venue: CHI
+venue_location: Yokohama, Japan
+venue_url: https://chi2025.acm.org
+type:
+  - Conference
+tags:
+  - Visualization
+  - Machine Learning
+  - Visual Analytics
+pdf: https://arxiv.org/pdf/2410.01088
+arxiv: https://arxiv.org/abs/2410.01088
```

Comment:
```suggestion
arxiv: https://arxiv.org/abs/2410.01088
html: https://arxiv.org/html/2410.01088v2
```

Reply:
Didn't know you could specify version with `v1` `v2` etc. Neat!

Reply:
Oh, looks like it's actually not needed anymore. IIRC a few months ago the HTML rendering only worked when you included it so I couldn't link to it automatically. 

Maybe I can simplify the code now.